src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -94,6 +94,7 @@
                     + "                                         ERROR,WARN,INFO,DEBUG,TRACE [default: INFO].\n"
                     + " -Q, --quickcheck-test-mode              Run in QuickCheck test mode\n"
                     + " -P <seconds>, --cm-poll-interval        Configuration manager poll interval [default: 1]\n"
+                    + " -v, --verify                            Compute and verify checksums for log entries.\n"
```

Comment:
I recommend avoiding this kind of policy decision at log unit start.  If the log unit has a large amount of data, where "large" is any resource that I don't want to use or allocate at startup time.  For example, a petabyte of storage is large when I don't want to consume the CPU & I/O bandwidth to verify checksums of all of that petabyte in the context of constructing a new LogUnitServer instance.

In a related policy question, what if you find one byte of corruption in that petabyte.  What do you do?  Will the constructor fail?  If the constructor will always fail, then how do you fix the error without an object instance?  Will you deny access to all of the uncorrupted (1PB - 1) bytes?